### DISCUSSION TITLE
Duplicate 'categories'

### DISCUSSION THREAD

**Discussion by GregLeonhardt** [ryanhowdy/fcms#645] [Ideas]
When uploading photos (advanced mode) the 'Category' box defaults to "Create New Category".  This is fine but when the user enters a category name that already exists, a second category is created with the exact same name.  This can be extremely confusing.

**Comment by ryanhowdy** [comment]
I'd consider this an enhancement.

During the upload process we should do a check that the category name is unique and if not it will ask you if you meant the other existing category.
